parser/float_operation.go

```diff
@@ -91,6 +91,8 @@ func ToFloat64(unk interface{}) float64 {
 		return float64(i)
 	case int:
 		return float64(i)
+	case int32:
```

Comment:
Acá convendría hacer esto en vez de tener tantos casos:
```suggestion
func ToFloat64(unk interface{}) float64 {
	switch i := unk.(type) {
	case int, float32, float64, int32, int64:
		return reflect.ValueOf(i).Float()
	default:
		return 0
	}
}

```